artemis-protocols/artemis-mqtt-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/mqtt/MQTTUtil.java

```diff
@@ -139,6 +139,8 @@ public class MQTTUtil {
 
    public static final int DEFAULT_MAXIMUM_PACKET_SIZE = MAX_PACKET_SIZE;
 
+   public static final int DEFAULT_MAXIMUM_IN_FLIGHT_PUBLISH_MESSAGES = -1; // -1 means no limit
```

Comment:
@jbertram What do you think about _small_  change of the current behavior - set `DEFAULT_MAXIMUM_IN_FLIGHT_PUBLISH_MESSAGES` to 65535 ?
In the MQTT protocol it is impossible to send more than 65535 unACKed PUBLISH.
Before ARTEMIS-5467 -> broker hangs
After ARTEMIS-5467 -> exception and connection closing.
So I think is is not a big deal.

Reply:
I think that's fine. Although I recommend using `MQTTUtil.TWO_BYTE_INT_MAX` instead of `65535`.